static/sass/_pattern_navigation.scss

```diff
@@ -4,7 +4,6 @@
   }
 
   .p-navigation .p-navigation__row {
```

Comment:
This was the culprit. So I removed this instead of overwriting the padding. So the logo should be aligned now and on smaller screens we should recover the side spacing